src/Raven.Server/Documents/Handlers/Processors/Attachments/AttachmentHandlerProcessorForGetAttachment.cs

```diff
@@ -63,16 +63,46 @@ protected override async ValueTask GetAttachmentAsync(DocumentsOperationContext
                 HttpContext.Response.Headers[Constants.Headers.AttachmentSize] = attachment.Size.ToString();
```

Comment:
When using range, do wee need to send a different size?

Reply:
Since this is a custom header used by the client to indicate the actual size of the attachment I would not return something different. We are adding a ContentLength Header with the returned body size as per the spec.